integration_tests/models/schema.yml

```diff
@@ -124,6 +124,25 @@ models:
           - uniques:
               tags: ["error_test", "regular_tests"]
 
+  - name: backfill_days_column_anomalies
+    config:
+      elementary:
+        timestamp_column: updated_at
+    columns:
+      - name: "min_length"
+        tests:
+          - elementary.column_anomalies:
```

Comment:
Should pass because.the anomaly is 6 days ago and backfill_days is 2 by default.